float32 thrust_cap
int32 rotor_id
---

